tests/src/test/scala/whisk/core/database/test/RemoveLogsTests.scala

```diff
@@ -0,0 +1,104 @@
+
+package whisk.core.database.test
+
+import java.io.File
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+import org.junit.runner.RunWith
+import org.scalatest.FlatSpec
+import org.scalatest.junit.JUnitRunner
+
+import common.StreamLogging
+import common.TestUtils
+import common.WhiskProperties
+import common.WskActorSystem
+import whisk.core.entity.ActivationId
+import whisk.core.entity.EntityName
+import whisk.core.entity.EntityPath
+import whisk.core.entity.Subject
+import whisk.core.entity.WhiskActivation
+import whisk.core.entity.ActivationLogs
+
+@RunWith(classOf[JUnitRunner])
+class RemoveLogsTests
+    extends FlatSpec
+    with DatabaseScriptTestUtils
+    with StreamLogging
+    with WskActorSystem {
+
+    val designDocPath = WhiskProperties.getFileRelativeToWhiskHome("ansible/files/logCleanup_design_document_for_activations_db.json").getAbsolutePath
+    val removeLogsToolPath = WhiskProperties.getFileRelativeToWhiskHome("tools/db/deleteLogsFromActivations.py").getAbsolutePath
+
+    /** Runs the clean up script to delete old activations */
+    def removeLogsTool(dbUrl: String, dbName: String, days: Int, docsPerRequest: Int = 20) = {
+        println(s"Running removeLogs tool: $dbUrl, $dbName, $days, $docsPerRequest")
+
+        val cmd = Seq(python, removeLogsToolPath, "--dbUrl", dbUrl, "--dbName", dbName, "--days", days.toString, "--docsPerRequest", docsPerRequest.toString)
+        val rr = TestUtils.runCmd(0, new File("."), cmd: _*)
+    }
+
+    behavior of "Activation Log Cleanup Skript"
+
+    it should "delete logs in old activation and keep log in new activation" in {
+        val dbName = dbPrefix + "test_log_cleanup_1"
+        val db = createDatabase(dbName, Some(designDocPath))
+
+        try {
+            // Create an old and a new activation
+            val oldActivation = WhiskActivation(
+                namespace = EntityPath("testns1"),
+                name = EntityName("testname1"),
+                subject = Subject("test-sub"),
+                activationId = ActivationId.apply(),
+                start = Instant.now.minus(2, ChronoUnit.DAYS),
+                end = Instant.now,
+                logs = ActivationLogs(Vector("first line", "second line")))
+
+            val newActivation = WhiskActivation(
+                namespace = EntityPath("testns1"),
+                name = EntityName("testname1"),
+                subject = Subject("test-sub"),
+                activationId = ActivationId.apply(),
+                start = Instant.now,
+                end = Instant.now,
+                logs = ActivationLogs(Vector("first line", "second line")))
+
+            db.putDoc(oldActivation.docid.asString, oldActivation.toJson).futureValue shouldBe 'right
+            db.putDoc(newActivation.docid.asString, newActivation.toJson).futureValue shouldBe 'right
+
+            // Run the tool to delete logs from activations that are older than 1 day
+            removeLogsTool(dbUrl, dbName, 1)
```

Comment:
The script does a check, if the activation is older than the given days (in milliseconds).
I'd like to avoid the tests of the edge cases. The reason is, that I think, that they will fail in the pipeline, if the test is a bit slower than expected.